integration_tests/samples/socket_mode/aiohttp_example.py

```diff
@@ -23,12 +23,12 @@ async def process(client: SocketModeClient, req: SocketModeRequest):
         if req.type == "events_api":
             response = SocketModeResponse(envelope_id=req.envelope_id)
             await client.send_socket_mode_response(response)
-
-            await client.web_client.reactions_add(
-                name="eyes",
-                channel=req.payload["event"]["channel"],
-                timestamp=req.payload["event"]["ts"],
-            )
+            if req.payload["event"]["type"] == "message":
```

Comment:
Fixed the issue where this example may fail if the app subscribes to other events like reaction_added